app/design/frontend/base/default/template/timedbanners/page/html/notices.phtml

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE_AFL.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magentocommerce.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magento to newer
+ * versions in the future. If you wish to customize Magento for your
+ * needs please refer to http://www.magentocommerce.com for more information.
+ *
+ * @category   design_default
+ * @package    Mage
+ * @copyright  Copyright (c) 2016 Aligent Consulting (http://www.aligent.com.au)
+ * @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ */
+
+/**
+ * @see Mage_Page_Block_Html_Notices
+ */
+$_helper = Mage::helper('aligent_timedbanners');
+?>
+<?php if ($this->displayNoscriptNotice()): ?>
+    <noscript>
+        <div class="noscript">
+            <div class="noscript-inner">
+                <p><strong><?php echo $this->__('We detected that your JavaScript seem to be disabled.'); ?></strong></p>
+                <p><?php echo $this->__('You must have JavaScript enabled in your browser to utilize the functionality of this website.'); ?></p>
+            </div>
+        </div>
+    </noscript>
+<?php endif; ?>
+<?php if ($this->displayDemoNotice()): ?>
+    <p class="demo-notice"><?php echo $this->__('This is a demo store. Any orders placed through this store will not be honored or fulfilled.') ?></p>
+<?php endif; ?>
+<?php if ($_helper->shouldShowBanner()): ?>
+    <p class="demo-notice timed-banner"><?php echo $_helper->getMessage() // already localised ?></p>
```

Comment:
I can't see this message being escaped either here or in the helper function.  Is allowing the admin to include HTML in the message a requirement, or should we be escaping html in the message?  Concerned that this might be used for XSS, low risk, but...